extensions/ql-vscode/package.json

```diff
@@ -1686,10 +1686,6 @@
           "command": "codeQL.mockGitHubApiServer.unloadScenario",
           "when": "config.codeQL.mockGitHubApiServer.enabled && codeQL.mockGitHubApiServer.scenarioLoaded"
         },
-        {
```

Comment:
Sorry why has this been removed?

Reply:
This section adds conditions for when the command should appear in the Command Palette. Since it should always appear now, I've removed the conditioning 🙌🏼

Reply:
Oh sorry, dumb moment! I forget it just shows up if you don't define it here 🤦‍♀️